uber/templates/groups/history.html

```diff
@@ -30,14 +30,36 @@
 {% endfor %}
 </table>
 
+    {% if group.stripe_transactions %}
```

Comment:
I am... not sure. In any case, yeah, if #2471 gets held up this should be excised. I was thinking we'd just wait to merge this until after that PR is merged.